.github/workflows/publish.yml

```diff
@@ -0,0 +1,142 @@
+# https://docs.github.com/en/actions/creating-actions/creating-a-composite-action
+name: Publish binaries
+
+on:
+  pull_request:
+    types: [opened, synchronize, closed]
+    branches:
+      - "main"
+      - "master"
+  workflow_dispatch:
+    inputs:
+      distro:
+        description: Distro
+        required: true
+        type: choice
+        options:
+          - debian
+          - alpine
+      arch:
+        description: Arch
+        required: true
+        type: choice
+        options:
+          - aarch64
+          - amd64
+          - armv6hf
+          - armv7hf
+          - i386
+      node-version:
+        description: Node version
+        required: true
+        type: string
+
+env:
+  DISTRO: ${{ inputs.distro || 'debian' }}
+  ARCH: ${{ inputs.arch || 'aarch64' }}
+  NODE_VERSION: ${{ inputs.node-version || '19.6.1' }}
+  BUCKET_NAME: ${{ vars.BUCKET_NAME || 'balena-packages' }}
```

Comment:
Should this be `resin-packages`? B/c atm I do see node 19.6.1 in the resin-packages bucket but not in `balena-packages`. And since we do have published balenalib images for node 19.6.1, I'm guessing the balenalib builder scripts are using resin-packages(?).

Reply:
Update: It looks like the resin-packages is defined in the contracts repo:
See: https://github.com/balena-io/contracts/blob/d06ad25196f67c4d20ad309941192fdddf80e307/contracts/sw.stack/node/contract.json#L102

So unless there is an extra automation that I'm missing, it sounds like we manually construct that url on PR time, so I expect that we could just start using balena-packages.